browser-switch/src/test/java/com/braintreepayments/browserswitch/BrowserSwitchClientTest.java

```diff
@@ -34,78 +30,63 @@
 
 public class BrowserSwitchClientTest {
```

Comment:
We will be renaming this class to `BrowserSwitchClientUnitTest` after this PR is merged.